users/bin/makemigrations

```diff
@@ -0,0 +1 @@
+poetry run alembic revision --autogenerate "$@"
```

Comment:
I'd make a Makefile or use pyinvoke :) Not a big fan of these scripts 😊

Reply:
agree :)

Reply:
I don't think we can use pyinvoke

I will have a look at a Makefile

Reply:
Changed to use `pyinvoke`, and to make it easier to write I did `make run`, so you can do

`make run migrate`
`make run makemigrations "Name migration"`
`make run server`